app/upgrades/v1/mainnet/finality_params.go

```diff
@@ -1,13 +1,11 @@
 package mainnet
 
-// TODO Some default parameters. Consider how to switch those depending on network:
-// mainnet, testnet, devnet etc.
 const FinalityParamStr = `{
-  "max_active_finality_providers": 100,
-  "signed_blocks_window": 100,
+  "max_active_finality_providers": 60,
+  "signed_blocks_window": 10000,
   "finality_sig_timeout": 3,
-  "min_signed_per_window": "0.1",
-  "min_pub_rand": 100,
-  "jail_duration": "86400s",
-  "finality_activation_height": 17500
+  "min_signed_per_window": "0.3",
+  "min_pub_rand": 8192,
+  "jail_duration": "600s",
+  "finality_activation_height": 17480
```

Comment:
What's the calculation behind this value?

Reply:
`finality_activation_height` value as `17480` is defined at the spec page as well

Reply:
I think we should already increase it by 20 based on some latest updates.